testcases/git_diff/duplicate-lines.in.html

```diff
@@ -1,6 +1,3 @@
-<!-- This test highlights broken formatting behavior: the second line is the one that gets added in
```

Comment:
Essentially I had to remove because these comments were added after git diff was calculated which was leading to a conflict with how the line counting algorithm functions essentially, if these comments were in the file and then we calculated git diff then the diff line counting would work as expected.

Reply:
Added back